src/pages/en/guides/cms/ghost.mdx

```diff
@@ -2,12 +2,287 @@
+
+## Making a blog with Astro and Ghost
+
+With the setup above, you are now able to create a blog that uses Ghost as the CMS.
+
+### Prerequisites
+
+1. A Ghost blog 
+2. An Astro project integrated with the [Ghost content API](https://www.npmjs.com/package/@tryghost/content-api) - See [integrating with Astro](/en/guides/cms/ghost/#integrating-with-astro) for more details on how to set up an Astro project with Ghost.
+
+This example will create an index page that lists posts with links to dynamically-generated individual post pages.
+
+### Fetching Data
+
+You can fetch your site's data with the Ghost content API package.
+
+First, create an `api.ts` file under a `lib` directory.
+
+<FileTree title="Project Structure">
+- src/
+  - lib/
+    - **api.ts**
+  - pages/
+    - index.astro
+- astro.config.mjs
+- package.json
+</FileTree>
+
+Initialize an API instance with the Ghost API using the API key from the Ghost dashboard's Integrations page.
+
+```ts title="src/lib/api.ts"
+
+import GhostContentAPI from '@tryghost/content-api';
+
+// Create API instance with site credentials
+export const api = new GhostContentAPI({
+    url: 'http://localhost:2368', // This is the URL if your site is running on a local environment
+    key: import.meta.env.CONTENT_API_KEY,
+    version: 'v5.0',
+});
+
+```
+
+
+### Displaying a list of posts
+
+The page `src/pages/index.astro` will display a list of posts, each with a description and link to its own page.
+
+<FileTree title="Project Structure">
+- src/
+ - lib/
+    - api.ts
+  - pages/
+    - **index.astro**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+Import the API in the Astro frontmatter to use the `posts.browse()` method to access blog posts from Ghost. Set `limit: all` to retrieve all posts.
+
+```astro title="src/pages/index.astro"
+---
+import { api } from '../lib/api';
+const posts = await api.posts
+    .browse({
+        limit: 'all',
+    })
+    .catch((err) => {
+        console.error(err);
+    });
+---
+```
+
+Fetching via the content API returns an array of objects containing the [properties for each post](https://ghost.org/docs/content-api/#posts) such as:
+- `title` - the title of the post
+- `html` - the HTML rendering of the content of the post
+- `feature_image` - the source URL of the featured image of the post
+- `slug` - the slug of the post
+
+Use the `posts` array returned from the fetch to display a list of blog posts on the page.
+
+```astro title="src/pages/index.astro"
+---
+import { api } from '../lib/api';
+const posts = await api.posts
+    .browse({
+        limit: 'all',
+    })
+    .catch((err) => {
+        console.error(err);
+    });
+---
+
+<html lang="en">
+    <head>
+        <title>Astro + Ghost 👻</title>
+    </head>
+    <body>
+
+        {
+            posts.map((post) => (
+                <>
+                    <a href={`/post/${post.slug}`}>
+                        <h1> {post.title} </h1>
+                    </a>
+                </>
```

Comment:
we can remove fragments here since they are not needed

```suggestion
                <a href={`/post/${post.slug}`}>
                    <h1> {post.title} </h1>
                </a>
```